src/App.css

```diff
@@ -9,15 +13,40 @@ body {
   background: linear-gradient(to right, rgb(58, 123, 213), rgb(58, 96, 115));
 }
 
+#root {
```

Comment:
Just wonder, why did you decide to use here instead of class? It 10 times stronger than class, why do you need it?

Reply:
I've just wanted to apply styles to the div that had only id, but now I see that I can move styles to body